Themes/default/index.template.php

```diff
@@ -200,24 +206,31 @@ functions on the SMF wiki.
  */
 function template_body_above()
 {
-	// Wrapper div now echoes permanently for better layout options. h1 a is now target for "Go up" links.
+
+	// Header
 	echo '
-	<div id="top_section">
-		<div class="inner_wrap">';
+	<div id="header">
+		<div class="inner_wrap">
+			<h1 class="forumtitle">
+				<a id="top" href="', Config::$scripturl, '">', empty(Utils::$context['header_logo_url_html_safe']) ? '<img id="smflogo" src="' . Theme::$current->settings['images_url'] . '/smflogo.svg" alt="Simple Machines Forum" title="Simple Machines Forum">' : '<img src="' . Utils::$context['header_logo_url_html_safe'] . '" alt="' . Utils::$context['forum_name_html_safe'] . '">', '</a>
+			</h1>';
 
+    //User Panel
 	// If the user is logged in, display some things that might be useful.
+	echo '
+	<div class="user_panel">';
 	if (User::$me->is_logged)
 	{
 		// Firstly, the user's menu
 		echo '
-			<ul class="floatleft" id="top_info">
+			<ul id="top_info">
 				<li>
 					<a href="', Config::$scripturl, '?action=profile"', !empty(Utils::$context['self_profile']) ? ' class="active"' : '', ' id="profile_menu_top">';
 
 		if (!empty(User::$me->avatar))
 			echo User::$me->avatar['image'];
 
-		echo '<span class="textmenu">', User::$me->name, '</span></a>
+		echo '</a>
```

Comment:
If we do this for the avatar image, shouldn't we use a alt text for the name?

Reply:
Should use href or url then.
Or should 'image' provide it by default? It would be good in case an external avatar is missing in other parts of the forum.

Reply:
I think either is valid.  I believe it just needs something to be valid for screen readers. Since its a link to the users profile and we removed the name, the alt should have it.

Reply:
Then I think 'image' should provide the alt text by default. An external avatar could suddenly not load or disappear, would be good to have it.